addons/website/controllers/form.py

```diff
@@ -279,6 +278,25 @@ def insert_record(self, request, model, values, custom, meta=None):
                     message_type='comment',
                 )
 
+        if hasattr(record, '_message_log'):
+            warning_icon = ""
+            authenticate_message = False
+            if request.session.uid:
+                model_record = request.env[model_name]
+                user_email = request.env.user.email
+                form_email = values[model_record._mail_get_primary_email_field()]
```

Comment:
Won't this crash if the primary_email_field is not in values? (could also return `None` as primary_email_field?)